pkg/controller/controller_instance_test.go

```diff
@@ -441,7 +499,7 @@ func TestReconcileInstance(t *testing.T) {
 
 	updatedInstance := assertUpdateStatus(t, actions[0], instance)
 	assertInstanceReadyTrue(t, updatedInstance)
-	assertInstanceDashboardURL(t, instance, testDashboardURL)
+	assertInstanceDashboardURL(t, updatedInstance, testDashboardURL)
```

Comment:
Should this have failed before?
Is this change due to a code change?
Was it because we weren't deep copying something we should have been?